spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowServerConfiguration.java

```diff
@@ -162,12 +164,28 @@ public Object postProcessAfterInitialization(Object bean, String s) throws Beans
 
 	@Bean
 	@ConditionalOnMissingBean
+	@ConditionalOnExpression("#{'${serverStore:redis}'.equals('db')}")
```

Comment:
No, this wasn't a copy-paste error ;) I was just providing the default value for `serverStore`. Eventually, this property itself will be removed.